app/src/main/java/org/mozilla/fenix/components/metrics/Event.kt

```diff
@@ -203,41 +202,6 @@ sealed class Event {
     object LoginDialogPromptSave : Event()
     object LoginDialogPromptNeverSave : Event()
 
-    // Tab tray
-    object TabsTrayOpened : Event()
-    object TabsTrayClosed : Event()
-    data class OpenedExistingTab(val source: String) : Event() {
-        override val extras = mapOf(TabsTray.openedExistingTabKeys.source to source)
-    }
-    data class ClosedExistingTab(val source: String) : Event() {
-        override val extras = mapOf(TabsTray.closedExistingTabKeys.source to source)
-    }
```

Comment:
@travis79 Does moving away from the deprecated `record(map) ` to `record(object) ` change something in the way the data is collected. Do we need to notify DS of such changes? Thank you!

Reply:
Great call to check on this! The data is actually in the same format for the purpose of analysis, so there shouldn't be any need to inform DS about updating away from the deprecated API.